modules/test/playwright/tests/headless-builder-web/schemas.spec.ts

```diff
@@ -275,32 +535,96 @@ testFeatureFlagsEnabled(
 		await headlessBuilderPage.goto();
 		await headlessBuilderPage.goToEditApplication(application.title);
 		await applicationPage.goToSchemasTab();
-		await applicationPage.addSchemaButton.click();
-		await applicationPage.schemaObjectDefinitionSelector.click();
+		await schemaPage.goTo('ObjectDefinition1 Schema');
+		await schemaPage.goToPropertiesTab();
+
+		// Assert that principal object properties are enabled
 
 		expect(
-			(
-				await applicationPage.page.getByRole('menu').allInnerTexts()
-			)[0].split('\n')
-		).toEqual([
-			'AccountEntry',
-			'User',
-			'APISort',
-			'APIFilter',
-			'APIEndpoint',
-			'APIProperty',
-			'APISchema',
-			'APIApplication',
-			'ObjectDefinition',
-		]);
+			await schemaPage.page.getByLabel('Add Author Property')
+		).toBeEnabled();
+
+		await schemaPage.page
+			.getByRole('button', {name: 'View Related Objects'})
+			.click();
+
+		// Assert that unmodifiable system object properties are disabled
+
+		await schemaPage.page
+			.getByRole('button', {name: 'Organization'})
+			.click();
+		expectElementToHaveClass(
+			await schemaPage.page
+				.getByRole('button', {name: 'Organization'})
+				.locator('..')
+				.getByLabel('Test Unmodifiable System Object')
+				.getByLabel('Add Author Property')
+				.getByText('Author'),
+			'disabled'
+		);
+
+		// Assert that unmodifiable allowed system object properties are disabled with FF
```

Comment:
is the comment right? I think should be enabled with FF

Reply:
The comment is wrong, the code checks the correct thing :)